src/test/java/io/confluent/examples/streams/GenericAvroIntegrationTest.java

```diff
@@ -51,18 +50,28 @@
  */
 public class GenericAvroIntegrationTest {
 
+  // Create a mocked schema registry for our serdes to use
```

Comment:
Nice! My one comment is that it seems a little awkward to be directly accessing the private static CLIENT in order to register schemas: did you consider adding a #register method to the interface?

Reply:
Yeah, you could use the method to get it. `register` doesn't work, though, because the avro serdes are going to make their own instances of this class. That's why the field has to be static, so that all instances get a reference to the same client.

In another discussion I had another idea about how to do this, which I think you might like better. I'll update the proposal.